src/i18n/locales/en.json

```diff
@@ -113,5 +113,24 @@
   "taskView.update.title": "Update Task with ID: { id }",
   "property.descripton": "Description",
   "proppertty.area": "Propperty Area",
-  "laoding": "Loading ..."
+  "laoding": "Loading ...",
+  "viewTitle": "Object Data View",
```

Comment:
Please prefix the properties with e.g. newPropertyButton.viewTitle. Because we have only one translation file for the whole application.